2021-07-28 12:51:11 UTC Syncing #12914531 0x2a1d…0755     0.20 blk/s   41.6 tx/s    3.0 Mgas/s      0+    5 Qed Ancient:#6996574 LI:#12914537   27/50 peers      5 MiB chain    1 MiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-07-28 12:51:16 UTC Syncing #12914531 0x2a1d…0755     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+    5 Qed Ancient:#6996574 LI:#12914537   27/50 peers      5 MiB chain    1 MiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-07-28 12:51:21 UTC Syncing #12914532 0x71dd…a26f     0.20 blk/s   49.6 tx/s    3.0 Mgas/s      0+    4 Qed Ancient:#6996575 LI:#12914537   27/50 peers      5 MiB chain  839 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-07-28 12:51:26 UTC Syncing #12914532 0x71dd…a26f     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+    4 Qed Ancient:#6996575 LI:#12914537   26/50 peers      5 MiB chain  839 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-07-28 12:51:56 UTC  Ancient:#6996579 AB:#6999918   26/50 peers      5 MiB chain  427 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-07-28 12:52:26 UTC  Ancient:#6996581 AB:#6999918   25/25 peers      5 MiB chain  428 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-07-28 12:52:40 UTC Imported #12914540 0x544d…ff22 (223 txs, 15.00 Mgas, 9097 ms, 69.41 KiB)
2021-07-28 12:52:56 UTC  Ancient:#6997354 AB:#7000659   27/50 peers      8 MiB chain  510 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-07-28 12:53:26 UTC  Ancient:#6997358 AB:#7000659   27/50 peers      5 MiB chain  0 bytes queue  RPC:  0 conn,    0 req/s,    0 µs
2021-07-28 12:53:32 UTC Imported #12914546 0xe877…a64f (128 txs, 14.99 Mgas, 7425 ms, 45.44 KiB)
2021-07-28 12:53:39 UTC Panic occured, see stderr for details


====================

   0: panic_hook::set_with::{{closure}}
   1: std::panicking::rust_panic_with_hook
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:595:17
   2: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:497:13
   3: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/sys_common/backtrace.rs:141:18
   4: rust_begin_unwind
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:493:5
   5: core::panicking::panic_fmt
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/panicking.rs:92:14
   6: core::option::expect_none_failed
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/option.rs:1300:5
   7: ethcore_blockchain::blockchain::BlockChain::prepare_update
   8: ethcore_blockchain::blockchain::BlockChain::insert_unordered_block
   9: std::sys_common::backtrace::__rust_begin_short_backtrace
  10: core::ops::function::FnOnce::call_once{{vtable.shim}}
  11: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/alloc/src/boxed.rs:1521:9
      <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/alloc/src/boxed.rs:1521:9
      std::sys::unix::thread::Thread::new::thread_start
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/sys/unix/thread.rs:71:17
  12: start_thread
  13: __clone


Thread 'ancient_block_exec' panicked at 'Low level database error when updating blooms. Some issue with disk?: Os { code: 11, kind: WouldBlock, message: "Resource temporarily unavailable" }', crates/ethcore/blockchain/src/blockchain.rs:1455

This is a bug. Please report it at:

    https://github.com/openethereum/openethereum/issues/new


2021-07-28 12:53:40 UTC Finishing work, please wait...
2021-07-28 12:53:40 UTC Panic occured, see stderr for details


====================

   0: panic_hook::set_with::{{closure}}
   1: std::panicking::rust_panic_with_hook
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:595:17
   2: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:497:13
   3: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/sys_common/backtrace.rs:141:18
   4: rust_begin_unwind
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:493:5
   5: core::panicking::panic_fmt
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/panicking.rs:92:14
   6: core::option::expect_none_failed
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/option.rs:1300:5
   7: ethcore::client::client::Client::shutdown
   8: openethereum::main
   9: std::sys_common::backtrace::__rust_begin_short_backtrace
  10: main
  11: __libc_start_main
  12: _start


Thread 'main' panicked at 'Join handle should not panic: Any', /home/longz/.cargo/git/checkouts/reth-1e6c266c722f7d6d/573e128/crates/util/src/queue/execution_queue.rs:59

This is a bug. Please report it at:

    https://github.com/openethereum/openethereum/issues/new